docs/DEVELOPER_NODE.md

```diff
@@ -0,0 +1,106 @@
+# AlastriaID Developer Node
+
+Alastria provides a node in the T (GoQuorum, IBFT 1.0) and B (BESU, IBTF 2.0) networks for those developers who want to contribute to the AlastriaID project.
+
+Access to this node is not open: access must be requested through this form.
+
+This guide describes how to access this resource:
+
+## Metamask
+
+Metamask is a very popular wallet that can be used for managing Ethereum accounts in a simple way.
+
+The connection with the nodes is made through the following URLs, using the access APIKEY:
+
+* AlastriaT Network:
+![AlastriaT Network](./img/alastria-t-metamask.jpg "AlastriaT Network")
+
+* AlastriaB Network:
+![AlastriaB Network](./img/alastria-b-metamask.jpg "AlastriaB Network")
+
+### Example
+
+![AlastriaB Network](./img/alastria-b-metamask-ok-0x50f606eabc06443cff710f9110d6c8ece8fd23206ac2d8adbcc6603d5828c204.jpg "AlastriaB Network")
+
+## Remix
+
+Remix is a well known development environment for Solidity that works both in the browser and on the desktop.
+
+The connection to the nodes can be made directly to the RPC service (http) of the node, or through Metamask
+
+There are some settings to consider for a successful connection:
+
+* Set EVM version (only for T Network):
+![AlastriaT Network](./img/alastria-t-remix-version.jpg "AlastriaT Network")
+
+* Use HTTP as Web3 provider:
+![AlastriaB Network](./img/alastria-t-remix-http-provider.jpg "AlastriaB Network")
+
+* Or use Metamask as provider:
+![AlastriaB Network](./img/alastria-t-remix-deploy-metamask.jpg "AlastriaB Network")
+
+## Truffle
+
+The connection to the node through Truffle is similar to those described above.
+
+These examples are based on the repository https://github.com/alastria/smart-contract-deployment:
+
+```
+ export SIGNER_PRIVATE_KEY=ae...
+ export T_NODE_IP=https://red-t.alastria.io/v0/94...
+ export B_NODE_IP=https://red-b.alastria.io/v0/94...
+```
+
+```
+npm install --save-dev truffle ganache-cli @truffle/hdwallet-provider dotenv
+```
+
+```
+truffle migrate --network alastriat                                                                                   INT ✘  at 00:09:05  
```

Comment:
at the end of the line there is some output from the terminal

Reply:
Thanks for the feedback: fixed!